### ISSUE TITLE
reportDir which is a path to report directory doesn't work

### ISSUE THREAD

**Issue by bkarakashev** [JakubRumpca/azure-pipeline-html-report#11]
When I specify the path to the report directory it says no `##[warning]Not found reportDir: /azp/agent/_work/10/....`
I have to set the exact path to the file in order to work.

**Comment by ryanpudd** [comment]
I see the same - looks like the commits on 8th Jan broke the directory functionality and it only supports single file now

**Comment by odecif** [comment]
Yes, same here. Would be really nice to figure out what caused this.

**Comment by dikuropiatnyk** [comment]
@JakubRumpca How hard would it be to implement the directory support?

**Comment by rubenmamo** [comment]
Same here

**Comment by mmaryo** [comment]
same

**Comment by RahulS249836** [comment]
Any update on this?

**Comment by RahulS249836** [comment]
@JakubRumpca

**Comment by techker** [comment]
just installed and have the same issue?is there any plan for this or is there a different way to fix this?
